app/src/main/java/org/wikipedia/util/DimenUtil.kt

```diff
@@ -42,9 +42,7 @@ object DimenUtil {
     }
 
     fun getFontSizeFromSp(window: Window, fontSp: Float): Float {
-        val metrics = DisplayMetrics()
-        window.windowManager.defaultDisplay.getMetrics(metrics)
-        return fontSp / metrics.scaledDensity
+        return fontSp / window.context.resources.displayMetrics.scaledDensity
```

Comment:
If we're already updating this code, can we just use the new (non-deprecated) suggested usage of `TypedValueCompat.applyDimension(...)` instead?

Reply:
Thanks! Done.